### DISCUSSION TITLE
Current Cover Art Url

### DISCUSSION THREAD

**Discussion by rxritalin** [music-assistant/support#2522] [Q&A] [answered]
Im playing around with using the Picture Elements card in combination with mini-media-player to try and create a UI I have in mind for a dedicated screen on my stereo setup. 

I there are url that points to currently playing album art? 
I've found this digging around in DevTools, but its seems the token changes with each image as it loads the next track. 
http://192.168.86.6:8123/api/media_player_proxy/media_player.squeeze_rx_pc?token=XXXXXXXX&cache=XXXXXXXX

Any help or is this a dead-end at present?

**Comment by OzGav** [comment, marked as answer]
Can you just reference to the attribute?

**Reply by rxritalin** [reply]
Yes, this works. Thank you. 
I was trying to use the attribute in the Picture Elements card, for the background image, but it does not seem to accept this. 
I found another work around though.
